tests/fast-integration/skr-svcat-migration-test/deploy-sample-resources.js

```diff
@@ -0,0 +1,212 @@
+  let t0 = performance.now()
+
+  options = options || {};
+  
+  const serviceInstanceManifestPath = options.resourcesPath || join(__dirname, "fixtures", "02_serviceinstance_html5appsrepo.yaml");
+  const functionManifestPath = options.resourcesPath || join(__dirname, "fixtures", "03_function_html5appsrepo.yaml");
+  const funcSBManifestPath = options.resourcesPath || join(__dirname, "fixtures", "04_func-sb_html5appsrepo.yaml");
+  const instSBManifestPath = options.resourcesPath || join(__dirname, "fixtures", "04_inst-sb_html5appsrepo.yaml");
+  const sbuManifestPath = options.resourcesPath || join(__dirname, "fixtures", "05_sbu_html5appsrepo.yaml");
+  
+  await installRessource(serviceInstanceManifestPath, "serviceInstance", "svcat-html5-apps-repo-1", "default")
+  await installRessource(functionManifestPath, "function", "svcat-html5-apps-repo-1", "default")
+  await installRessource(funcSBManifestPath, "servicebinding", "func-sb-svcat-html5-apps-repo-1", "default")
+  await installRessource(instSBManifestPath, "servicebinding", "inst-sb-svcat-html5-apps-repo-1", "default")
+  await installRessource(sbuManifestPath, "servicebindingusage", "func-sbu-svcat-html5-apps-repo-1", "default")
+  
+  // return console.timeEnd('installHTML5AppsRepoExample')
+  let t1 = performance.now()
+  console.log(`Finished deployment of HTML5-Apps-Repo in ${t1-t0} ms`)
+  return {ressource: "HTML5-Apps-Repo", duration: t1-t0}
+}
+
+async function installAuditManagementExample(options) {
+  // console.time('installAuditManagementExample')
+  let t0 = performance.now()
+
+  options = options || {};
+  
+  const serviceInstanceManifestPath = options.resourcesPath || join(__dirname, "fixtures", "02_serviceinstance_auditlogmanagement.yaml");
+  const functionManifestPath = options.resourcesPath || join(__dirname, "fixtures", "03_function_auditlogmanagement.yaml");
+  const funcSBManifestPath = options.resourcesPath || join(__dirname, "fixtures", "04_func-sb_auditlogmanagement.yaml");
+  const instSBManifestPath = options.resourcesPath || join(__dirname, "fixtures", "04_inst-sb_auditlogmanagement.yaml");
+  const sbuManifestPath = options.resourcesPath || join(__dirname, "fixtures", "05_sbu_auditlogmanagement.yaml");
+  
+  await installRessource(serviceInstanceManifestPath, "serviceInstance", "svcat-auditlog-management-1", "default")
+  await installRessource(functionManifestPath, "function", "svcat-auditlog-management-1", "default")
+  await installRessource(funcSBManifestPath, "servicebinding", "func-sb-svcat-auditlog-management-1", "default")
+  await installRessource(instSBManifestPath, "servicebinding", "inst-sb-svcat-auditlog-management-1", "default")
+  await installRessource(sbuManifestPath, "servicebindingusage", "func-sbu-svcat-auditlog-management-1", "default")
+  
+  // return console.timeEnd('installAuditManagementExample')
+  let t1 = performance.now()
+  console.log(`Finished deployment of Audit-Management in ${t1-t0} ms`)
```

Comment:
this is pretty good idea, the testing framework does output elapsed time for each `It()` block that is over 5000ms.
```
  SKR SVCAT migration test
    ✓ Should provision SKR (1711998ms)
    ✓ Should save kubeconfig
    ✓ Should initialize K8s
    ✓ Should instantiate SM Instance and Binding (7749ms)
    ✓ Should install BTP Operator helm chart (22789ms)
    ✓ Should deprovision SKR (2371940ms)
    ✓ Should cleanup SM instances and bindings
```
But I like your approach and I agree we may want to have some more detailed and better consumable metrics to compose statistics than the default ms for slow `It()` blocks.